bin/tx-prover/src/proxy/mod.rs

```diff
@@ -412,6 +460,7 @@ impl ProxyHttp for LoadBalancer {
 
         // Check if the queue is full
         if queue_len >= self.0.max_queue_items {
+            QUEUE_DROP_COUNT.inc();
```

Comment:
Could we move this to the `create_queue_full_response()` function? And maybe the ones for the other types of repsonses as well

Reply:
Yeah, that sounds nice.

Reply:
Oh, I was moving it and saw that this will require to expose the metrics module for the `util.rs` in the `lib.rs` which will require add some feature check to not compile it if we are in WASM. I see two options:
1. Using the feature over the module exposure.
2. Leaving the metrics as they are now.

I have a soft preference for the second, but not 100% sure.